### PR TITLE
Add IsComputed  helper for field

### PR DESCRIPTION
**Pull request by pratikprasad**

### CODE REVIEW

**Comment by alexandrinaw** [comment]
Could you please add tests?

Could you also add more of a description to the PR? I'm struggling to figure out what this is for or how it would be useful.

**Comment by pratikprasad** [comment]
Added a better commit message and a test. The only real use case is so that I can use `Unpack` for structs that have computed fields. Unpack lives in the other repo - and we're currently not able to skip over copying fields that are computed since that information is encapsulated.

File: computed_by_test.go [hunk 1]
```diff
@@ -16,12 +16,24 @@ import (
...
 	"math"
 	"strings"
+	"testing"
 
 	"github.com/helloeave/dat/sqlx-runner"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
+func TestField_IsComputedBy(t *testing.T) {
```
Comment (alexandrinaw) [inline computed_by_test.go]: Can you make this part of the `Zuite`

File: computed_by_test.go [hunk 2]
```diff
@@ -14,24 +14,22 @@ package worksheets
...
 	"fmt"
-	"math"
-	"strings"
-	"testing"
-
 	"github.com/helloeave/dat/sqlx-runner"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"math"
+	"strings"
```
Comment (alexandrinaw) [inline computed_by_test.go]: I feel like these are in the wrong spot now?

Comment (pratikprasad) [reply computed_by_test.go]: 😬 Sorry, I should have just run goimports...

**Comment by lgonzalez-silen** [comment]
> @pratikprasad you can't skip by specifying `ws:"-"`?

nvm @alexandrinaw explained to me that this is in the other conversion direction. let ws do its thing basically, chuck the struct's value.

**Pull request merged**